cfme/utils/appliance/__init__.py

```diff
@@ -1786,6 +1802,24 @@ def is_embedded_ansible_role_enabled(self):
     def is_embedded_ansible_running(self):
         return self.is_embedded_ansible_role_enabled and self.is_supervisord_running
 
+    @property
+    def is_failover_started(self):
+        log = ('/var/www/miq/vmdb/log/ha_admin.log' if self.version < 5.10
+               else '/var/www/miq/vmdb/log/evm.log')
+        return self.ssh_client.run_command(
```

Comment:
Remove ```<5.10``` checks.

Reply:
I don't want to do that right now. Maybe in other commit. Reasons above.